src/SprykerEco/Yves/Adyen/Handler/Redirect/KlarnaInvoiceRedirectHandler.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Adyen\Handler\Redirect;
+
+use Generated\Shared\Transfer\AdyenRedirectResponseTransfer;
+
+class KlarnaInvoiceRedirectHandler extends OnlineTransferRedirectHandler
```

Comment:
Is klarna now part of online transfer payment method?

Reply:
Yes, it is